File: 8KYU/Training JS #32: methods of Math---round() ceil() and floor()/solution.py
import math
def round_it(n):
    print(n)
    splits = str(n).split('.')
    if len(splits[0]) > len(splits[1]):
        return math.floor(n)
    elif len(splits[0]) < len(splits[1]):
        return math.ceil(n)
    return round(n)

File: 8KYU/Training JS #32: methods of Math---round() ceil() and floor()/solution.js
function roundIt(n){
    var digits = n.toString().split('.');
    if(digits[0].length > digits[1].length)
      return Math.floor(n);
    else if(digits[0].length < digits[1].length)
      return Math.ceil(n);
    return Math.round(n);
}